BaragonAgentService/src/main/java/com/hubspot/baragon/agent/handlebars/PreferSameRackWeightingHelper.java

```diff
@@ -18,7 +22,7 @@ public PreferSameRackWeightingHelper(BaragonAgentConfiguration configuration, Ba
     this.agentMetadata = agentMetadata;
   }
 
-  public CharSequence preferSameRackWeighting(Collection<UpstreamInfo> upstreams, UpstreamInfo currentUpstream, Options options) {
+  public CharSequence preferSameRackWeightingOriginal(Collection<UpstreamInfo> upstreams, UpstreamInfo currentUpstream, Options options) {
```

Comment:
It's fine to remove this method (and its associated tests), since it's all in version control if we need to bring it back